src/styles.css

```diff
@@ -38,14 +38,17 @@ body {
 }
 
 .board-cell.sprite-1 {
+  animation: scale-up-center 1.5s linear infinite alternate-reverse both;
   background-image: url('/images/sweet-1.png');
 }
 
 .board-cell.sprite-2 {
+  animation: scale-up-center 1.5s linear 0.5s infinite alternate-reverse both;
```

Comment:
Instead of repeating the same line three times, do this

```css
.board-cell {
  animation: scale-up-center 1.5s linear infinite alternate-reverse both;
}

board-cell.sprite-2 {
  animation-delay: 0.5s;
}
```

Refer [this](https://www.w3schools.com/cssref/css3_pr_animation-delay.asp) article

Reply:
Will do